### ISSUE TITLE
[Accessibility]Remove an item from PeoplePicker doesn't announce anything

### ISSUE THREAD

**Issue by tianbli** [microsoft/fluentui#11467]
### Environment Information

- **Package version(s)**: 7.65.1
- **Browser and OS versions**: Narrator on Edge, NVDA on Chrome

### Describe the issue:
On normal People Picker, when no item has focused and cursor placed at the end of input, press backspace will remove the last item. But unlike other input box, screen reader doesn't announce which one is removed. Especially when there is only one item, remove the only one by placing cursor behind the item and pressing backspace won't announce anything. 

But when we use arrow key to select an item first then delete it, it will announce all of the remaining items.

### Please provide a reproduction of the issue in a codepen:
1. Go to https://codepen.io/tianbli/pen/povEbQa?editors=1111, make sure normal type is used
2. Select anyone from suggestion dropdown, then cursor will be behind the item. Press backspace to delete the item. Notice nothing is announced.

#### Actual behavior:
Nothing is announced when the only one item is removed from PeoplePicker

#### Expected behavior:
Not sure exactly, but I expect the removed item should be announced just like how TextField does when use backspace to remove a character? Or announce all of the remaining items?

### Documentation describing expected behavior
N/A

**Comment by joschect** [comment]
@tianbli Thanks for submitting this! I know that this worked at one point, but since there is no direct aria guidance about this it's hard to know if it will continue to work. @jamesedjac and I are working on a way to get this read out without needing to force it with something like an alert.
